modules/stochastic_tools/test/tests/multiapps/batch_commandline_control/master_vector.i

```diff
@@ -0,0 +1,100 @@
+[StochasticTools]
+[]
+
+[Distributions]
+  [uniform]
+    type = Uniform
+    lower_bound = 5
+    upper_bound = 10
+  []
+[]
+
+[Samplers]
+  [sample]
+    type = MonteCarlo
+    num_rows = 3
+    distributions = 'uniform uniform uniform uniform'
+    execute_on = 'PRE_MULTIAPP_SETUP'
+  []
+[]
+
+[MultiApps]
+  [sub]
+    type = SamplerFullSolveMultiApp
+    sampler = sample
+    input_files = 'sub.i'
+    mode = batch-reset
+  []
+[]
+
+[Transfers]
+  [data]
+    type = SamplerPostprocessorTransfer
+    multi_app = sub
+    sampler = sample
+    to_vector_postprocessor = storage
+    from_postprocessor = size
+  []
+  [prop_A]
+    type = SamplerPostprocessorTransfer
+    multi_app = sub
+    sampler = sample
+    to_vector_postprocessor = prop_A
+    from_postprocessor = prop_A
+  []
+  [prop_B]
+    type = SamplerPostprocessorTransfer
+    multi_app = sub
+    sampler = sample
+    to_vector_postprocessor = prop_B
+    from_postprocessor = prop_B
+  []
+  [prop_C]
+    type = SamplerPostprocessorTransfer
+    multi_app = sub
+    sampler = sample
+    to_vector_postprocessor = prop_C
+    from_postprocessor = prop_C
+  []
+  [prop_D]
+    type = SamplerPostprocessorTransfer
+    multi_app = sub
+    sampler = sample
+    to_vector_postprocessor = prop_D
+    from_postprocessor = prop_D
+  []
+[]
+
+[VectorPostprocessors]
+  [storage]
+    type = StochasticResults
+  []
+  [prop_A]
+    type = StochasticResults
+  []
+  [prop_B]
+    type = StochasticResults
+  []
+  [prop_C]
+    type = StochasticResults
+  []
+  [prop_D]
+    type = StochasticResults
+  []
+[]
+
+[Controls]
+  [cmdline]
+    type = MultiAppCommandLineControl
+    multi_app = sub
+    sampler = sample
+    param_names = 'Mesh/xmax Materials/const/prop_values/[0,(1.5),1,1] Mesh/ymax'
```

Comment:
After our discussion, @jiangwen84 and I agreed that this approach would work with one exception. Change the syntax to the following, where the [] is the global column index from the sampler. By default, if the [] is not provided it is assumed that values are scalar and captured in ordered.

```suggestion
    param_names = 'Mesh/xmax[0] Materials/const/prop_values[2,(1.5),3,3] Mesh/ymax[1]'
```